storage/s3.go

```diff
@@ -620,6 +621,9 @@ func (s *S3) Get(
 	if from.VersionID != "" {
 		input.VersionId = aws.String(from.VersionID)
 	}
+	if contentRange != nil {
+		input.Range = aws.String(*contentRange)
+	}
 
 	return s.downloader.DownloadWithContext(ctx, to, input, func(u *s3manager.Downloader) {
```

Comment:
Do you think I should use the `Download()` function from your screenshot instead of `s.downloader.DownloadWithContext(...)`? Since it does the same thing as the code in line 624 of s3.go?